README.md

```diff
@@ -1,2 +1,124 @@
-# mv_hello-microverse
-Hello Microverse project. First project to setup a simple web page.
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [mv_hello-microverse] <a name="about-project"></a>
+
+mv_hello-microverse project!
+
+**mv_hello-microverse** It's a setup project for a simple hello webpage!
+
+## 🛠️ Built With <a name="built-with">
+### Tech Stack <a name="tech-stack"></a>
+- HTML
+- CSS
+
+</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Good HTML and CSS practices**
+- **A wepage with use of linters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A modern web browser like Google Chrome, Mozilla Firefox, Apple Safari or Microsoft Edge.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Clone the project using by running the following command :
+
+`git clone`
+`git@github.com:jcmunav63/mv_hello-microverse.git`
+
+
+
```

Comment:
- [ ] To make your README to be more professional and descriptive to non-technical users, I think it would be better if you improve information from the `Getting Started` section by adding some like how to `access` the cloned directory, `install` dependencies(such for linters), `use/run` your application locally, `run tests`(your linters tests),... I recommend you follow this **[Gist template](https://gist.github.com/tresorsawasawa/9e65422376a395c17d79bcb10c31077e)** as an example to help you complete everything.